editor/editor_settings.cpp

```diff
@@ -959,7 +959,7 @@ void EditorSettings::setup_network() {
 			continue;
 		}
 		// Select current IP (found)
-		if (ip == current) {
+		if (ip == IPAddress(current)) {
```

Comment:
Since it's cast and checked multiple times, it's probably better to do: `String ip_string = String(ip);` one in the beginning of the loop and use it for all checks and `hint +=`.

Reply:
> Since it's cast and checked multiple times, it's probably better to do: `String ip_string = String(ip);` one in the beginning of the loop and use it for all checks and `hint +=`.

@bruvzg  Should we keep this PR about the mechanics of being C++20 compatible only and make a subsequent PR that changes the behavior as requested? Maybe make a link from that PR to here; or proceed with making the change here?